### ISSUE TITLE
[FEATURE EMBER_DEFAULT_HELPER_MANAGER] Shipping

### ISSUE THREAD

**Issue by chancancode** [emberjs/ember.js#20054]
* [x] [RFC](https://github.com/emberjs/rfcs/pull/756)
  * [x] [Update](https://github.com/emberjs/rfcs/pull/788) @chancancode
* [x] glimmer-vm
  * [x] https://github.com/glimmerjs/glimmer-vm/pull/1348
  * [x] https://github.com/glimmerjs/glimmer-vm/pull/1384
  * [x] https://github.com/glimmerjs/glimmer-vm/pull/1389)
* [x] ember.js (https://github.com/emberjs/ember.js/pull/20052)
* [x] enable feature
* [x] [update guides](https://github.com/emberjs/rfcs/blob/master/text/0756-helper-default-manager.md#how-we-teach-this) (https://github.com/ember-learn/guides-source/pull/1924)
* [x] adopt polyfill into [ember-polyfills](https://github.com/ember-polyfills)
  * [x] update polyfill to be inert as of v4.x? (TBD)
* [x] coordinate with TypeScript/Glint team cc @chriskrycho @dfreeman @jamescdavis

**Comment by chancancode** [comment]
cc @Windvis @NullVoxPopuli I added it to the agenda for friday

**Comment by chriskrycho** [comment]
Glint is ready. 👍🏼

**Comment by NullVoxPopuli** [comment]
polyfill is moved to ember-polyfills, and makes itself inert as of 4.5.0-alpha.4

**Comment by chriskrycho** [comment]
@chancancode is this landing in v4.5? (Checking so we can get it in the blog post if so; cc. @jenweber).

**Comment by NullVoxPopuli** [comment]
Obviously, I'm not chancancode, but (I just happened to be looking at my email, and saw the release):

it did land -- here: https://github.com/emberjs/ember.js/releases/tag/v4.5.0


I even used the feature in an unrelated bug report! :partying_face:

**Comment by Windvis** [comment]
I think this can be closed now since the guides PR has been merged?
